x-pack/plugins/observability_solution/synthetics/tsconfig.json

```diff
@@ -105,6 +105,7 @@
     "@kbn/ebt-tools",
     "@kbn/alerting-types",
     "@kbn/core-chrome-browser",
+    "@kbn/core-rendering-browser-internal",
```

Comment:
Core's `*-internal` packages are not supposed to be imported outside core. I would suggest moving the helper to a "non-internal" package.

@gsoldevila, do you know if this will be limited by the project you're working on?

Reply:
@afharo Do you have a suggestion for where to put `useAppFixedViewport`?

Reply:
Not really... I don't know if there's an existing package that could be a good candidate. If unclear, I'd be OK with creating a new sibling package `@kbn/core-rendering-browser` inside `packages/core/rendering` that exports publicly importable utils.

Reply:
>  I'd be OK with creating a new sibling package @kbn/core-rendering-browser inside packages/core/rendering that exports publicly importable utils.

To be clear, core's public packages export public types that may be consumed by plugins.

Reply:
I agree with @afharo to create a new `@kbn/core-rendering-browser` package that (for now) solely contains the `useAppFixedViewport` hook is the best solution to follow existing precedent.

Reply:
Resolved in de0770d05cacc936d198867ba411da421facd5e5